src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>API: Leaderboard</title>
+</head>
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+
+  <main class="container">
+    <section class="scores-section">
+      <div class="recent">
+        <div class="sub-header">Recent scores</div>
+        <div>
+          <button class="btn refresh-btn">Refresh</button>
+        </div>
+      </div>
+
+      <div class="scores-list">
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+      </div>
+    </section>
+
+    <section class="input-section">
+      <div class="sub-header">Add your scores</div>
+
+      <form>
+        <input type="text" placeholder="Your name">
+        <input type="text" placeholder="Your score">
+        <div>
+          <button class="btn submit-btn" type="button">Submit</button>
+        </div>
+      </form>
+    </section>
+  </main>
+</body>
```

Comment:
- [ ] Index main contain a root element like `<div class="root"></div>`. This will help js code to know where this could run. Please, consider adding it on index.html and test that the js code understands that needs to work on that part. 🦊